system/openrc/openrc.SlackBuild

```diff
@@ -82,11 +82,24 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
+# Use a local Meson >= 0.62 just for this build
+VENV="$TMP/meson-venv"
+if [ ! -d "$VENV" ]; then
+  python3 -m venv "$VENV"
+  . "$VENV/bin/activate"
+  pip install --upgrade pip
+  pip install "meson==0.62.0" "ninja>=1.10"
```

Comment:
i mean that you should use meson and ninja from system packages, not via pip because it's not trackable via pkgtool

Reply:
This is the old commit. In the latest commit, I deleted this part entirely.